index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title></title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div id="main">
+      <div id="intro">
+        <div id="intro2">
```

Comment:
Naming could have been better instead of intro 2

Reply:
This request lacks a "why". Can you please explain to the author why the change was requested? 
For ex: _"Naming could have been better instead of intro 2 to make it more readable and understandable"_